### ISSUE TITLE
omdb api not longer working without paid API keys 

### ISSUE THREAD

**Issue by rolinux** [eefret/gomdb#11]
The library not working until we add support for that paid API keys.

[omdb api announcement of going private](https://www.patreon.com/posts/api-is-going-10743518)

**Comment by theapache64** [comment]
Yes, OMDB is not free anymore .Try something better than OMDB,  http://github.com/theapache64/movie_db
